node/src/accountant/mod.rs

```diff
@@ -881,21 +1025,21 @@ impl Accountant {
     }
 
     fn separate_early_errors(
-        sent_payments: SentPayable,
+        sent_payments: &SentPayable,
```

Comment:
I cannot say I'm particularly happy about the added references but I think I know why you did that at multiple places. Or I hope I do.